paasta_tools/secret_providers/vault.py

```diff
@@ -155,6 +155,14 @@ def get_secret_signature_from_data(self, data: Mapping[str, Any]) -> Optional[st
         else:
             return None
 
+    def get_secrets_from_vault_path(self, path=""):
```

Comment:
i wonder if this should be `get_data_from_vault_path` just to ensure that folks don't try to use this directly (they still need to wrap this in a k8s secret) and future users of this might just have a single piece of data at a given path  (although, there's some existing vault method that mention `secret`, so maybe `get_secret_from_vault_path`?)

Reply:
(and sidenote: this is missing type annotations)

Reply:
++ will make the change